storybook/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -128,3 +143,86 @@ export const WithWideContent: Story = {
     ],
   },
 }
+
+export const WithForm: Story = {
```

Comment:
You wanted to remove this before merge, right?

Reply:
Thanks for the heads up